use-cases/parser_chaining/README.md

```diff
@@ -233,3 +233,10 @@ cat ~/data.log | /usr/hdp/current/kafka-broker/bin/kafka-console-producer.sh --b
 ```
 
 You should see indices created for the `cisco-5-304` and `cisco-6-302` data with appropriate fields created for each type.
+
+# Aggregated Parsers with Parser Chaining
+Chained parsers can be run as aggregated parsers. These parsers continue to use the sensor specific Kafka topics, and do not do internal routing to the appropriate sensor.
+
```

Comment:
Should we explicitly say that when you use this type of topology so do NOT create a sensor topology for any sensor included in the aggregate?

Reply:
yes.  I'm not sure we need to document the UI or somewhere else?   Do we have to document removing the default processors from ambari if you are going to aggregate them etc?  Or how to do it in the management ui?

We may need more practical steps.

Reply:
Right now, as noted in the description, there's no UI attached to this.  Even the REST API's update is pretty minimal (just to take comma separated lists).  I didn't want to build that out, because the management UI requires some decent amount of thought put into it and that'll ripple through REST as needed (e.g. needing/wanting to pass spout num tasks, parallelism, etc.).

Right now I look at this as providing a low level way of being able to get some of the benefits of this type of aggregation, with making it more user friendly being follow-on since it'll require nontrivial effort and design. I can go ahead and create follow-on tickets for that work, if that works for you.

For the default Ambari processors, I'm not particularly inclined to worry about it, although I could be persuaded that we need to.  That feels like something that can be addressed as this is made more user friendly (i.e. I expect people familiar enough with the system to make the determination to aggregate parsers right now to also be familiar enough to stop the topologies).  I could add a warning or something like that in the docs to not run an aggregated parser with sensor X alongside a dedicated topology for sensor X, but I'm not sure that's necessary.

I also went ahead and added the actual command to the chain parsers README, so the practical example is complete.

Reply:
I think adding the follow on is a great idea.  I wonder if we shouldn't change the default install to aggregate the default sensors?  I don't think that will work though, because the comma separation in ambari is the list, and you won't be able to do it.

My main concern is that the reviewers and committers of this pr are going to be the only ones who can do it, and everyone in user land is going to be lost.  If this is going to be expert only ( until the UI comes ) and not recommend, or a preview thing, maybe mark it as such.

Reply:
I think we should adjust, as a follow-on, ambari to accept proper CSV for parsers.  So, if you want groups you just quote the groups.  E.g. let's say you input the following: `bro,"snort,yaf"`
That would create the following topologies:
* `bro`
* `snort` and `yaf` aggregated